bokeh/sphinxext/bokeh_github.py

```diff
@@ -34,7 +34,7 @@
 
 BOKEH_GH = "https://github.com/bokeh/bokeh"
 
-def bokeh_commit(name, rawtext, text, lineno, inliner, options={}, content=[]):
+def bokeh_commit(name, rawtext, text, lineno, inliner, options=None, content=[]):
```

Comment:
true, but it seemed to me this PR was about the principal of avoiding potential mistakes. We have to leave the function sig (it's what sphinx expects) and perhaps `content` will be used in the future, so why not change it now?